hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
OK, I just updated the PR, the key change is to have `MonitorServiceRestartParentAttributeHandler.applyUpdateToRuntime` always return `false`.  By doing this a config update can basically allow or not allow restart.  Either way the update will never set the server to 'reload-required'.  So, by default we don't restart, only if you explicitly add the option to allow restarts.  If you still feel we need to allow restart by default then we'll need some more work here.  Let me know what you think.